devices.js

```diff
@@ -1102,6 +1102,32 @@ const devices = [
         toZigbee: [],
         exposes: [],
     },
+    {
+        fingerprint: [{modelID: 'TS0101', manufacturerName: '_TYZB01_ijihzffk'}],
+        model: 'TS0101',
+        vendor: 'TuYa',
+        description: 'Zigbee Socket',
+        whiteLabel: [{vendor: 'Larkkey', model: 'PS080'}],
+        extend: preset.switch,
+        meta: {disableDefaultResponse: true},
```

Comment:
Last question before merge, is `disableDefaultResponse: true` really needed? (do you get timeout errors otherwise when sending state: ON?

Reply:
Yes, really needed! I confirmed for 2 samples, all need this.